egs/multi_zh-hans/ASR/zipformer/ctc_decode.py

```diff
@@ -0,0 +1,625 @@
+#!/usr/bin/env python3
```

Comment:
Do you have any RESULTS for CTC decoding? If yes, could you please post them in the RESULTS.md?

Reply:
the latest commit provides results of CTC decoding, i'm working on the related hugging face repo, it should be ready tonight